### ISSUE TITLE
SDK for adding your own colors to color scheme editor

### ISSUE THREAD

**Issue by Defman21** [Komodo/KomodoEdit#1649]
### Short Summary

title :stuck_out_tongue:

**Comment by cgchoffman** [comment]
You can pick any colour you want.  I don't understand what this is asking for.  Do you mean to add the ability to define new elements that can have a colour assigned to it?

**Comment by Defman21** [comment]
Yes, so the user can edit it via color scheme editor. Adding categories would be useful as well.

**Comment by Defman21** [comment]
And it will be extremely helpful if the colors added via color scheme editor for custom categories would be saved to `%profile_dir%/<category_name>.less` and somewhere in Komodo so I can access it via JS as well.

**Comment by Naatan** [comment]
You can already access it via JS: 

https://github.com/Komodo/KomodoEdit/blob/master/src/chrome/komodo/content/sdk/colorscheme.js#L96

**Comment by Defman21** [comment]
If this will work for custom categories that will be registered using the API I want, then ok :)

**Comment by Naatan** [comment]
That's the idea.
